ethcore/snapshot/snapshot-tests/src/helpers.rs

```diff
@@ -22,6 +22,18 @@ use std::sync::atomic::AtomicBool;
 
 use keccak_hash::{KECCAK_NULL_RLP};
 use account_db::AccountDBMut;
+<<<<<<< HEAD:ethcore/snapshot/snapshot-tests/src/helpers.rs
+use account_state;
+use blockchain::{BlockChain, BlockChainDB};
+use client_traits::ChainInfo;
+use common_types::{
+	ids::BlockId,
+	basic_account::BasicAccount,
+	errors::EthcoreError
+};
+use engine::Engine;
+use ethcore::client::Client;
+=======
```

Comment:
there are some merge conflicts here and the tests are passing, which is alarming